3-reveal/migrations/5-IRS/2-Zambia-2019/deploy/zambia_focus_area_irs.psql

```diff
@@ -54,11 +51,16 @@ SELECT * FROM
             WHEN irs_structures.foundStruct = 0 THEN 0
             ELSE CAST(irs_structures.sprayedStruct AS DECIMAL(7,2))/CAST(irs_structures.foundStruct AS DECIMAL(7,2))
         END AS spraySuccess,
+        (CAST(irs_structures.totStruct AS DECIMAL(7,2)) * 0.9) - CAST(irs_structures.sprayedStruct AS DECIMAL(7,2)) AS structures_remaining_to_90_se,
         CASE
             WHEN irs_structures.sprayed_rooms_eligible = 0 THEN 0
             ELSE CAST(irs_structures.sprayed_rooms_sprayed AS DECIMAL(7,2))/CAST(irs_structures.sprayed_rooms_eligible AS DECIMAL(7,2))
-        END AS roomCov
-        -- TODO: Mob
+        END AS roomCov,
+        CASE
+            WHEN coverage_query.sprayCov = 0 THEN 'n/a'
```

Comment:
looks like sprayCov needs an index

Reply:
`sprayCov` is calculated elsewhere in this query so we can't index it :-/

This is the relevant part:

```sql
    LEFT JOIN LATERAL (
        SELECT
            CASE
                WHEN irs_structures.totStruct = 0 THEN 0
                ELSE CAST(irs_structures.sprayedStruct AS DECIMAL(7,2))/CAST(irs_structures.totStruct AS DECIMAL(7,2))
            END AS sprayCov
    ) AS coverage_query ON true
```

Reply:
ah right, and we seems to rely on it later in this query so we need to calculate it